.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: java
+
+branches:
+  only:
+    - master
+
+before_cache:
+- rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+- rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+
+cache:
+  directories:
+  - $HOME/.gradle/caches/
+  - $HOME/.gradle/wrapper/
+
+before_install:
+  - curl https://raw.githubusercontent.com/hmcts/reform-api-docs/master/bin/publish-swagger-docs.sh > publish-swagger-docs.sh
+
+script:
+  - ./gradlew test
+  - ./gradlew integration
```

Comment:
can be added later ?

Reply:
The task exists already, so I think it's fine that it's included. We don't risk forgetting to add it later.